package.json

```diff
@@ -1,11 +1,40 @@
 {
-  "name": "codemods",
-  "version": "1.0.0",
-  "description": "Codemods for DHIS2 libs",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "Viktor Varland",
-  "license": "BSD-3-Clause"
+    "name": "@dhis2/cli-utils-codemods",
```

Comment:
We should change the repository name to match this as well

Reply:
@varl I don't have permissions do to that (in any repo), could you take care of that?

Reply:
Sure, done.